go.mod

```diff
@@ -7,12 +7,13 @@ require (
 	github.com/alecthomas/kong v0.2.17
 	github.com/crossplane/crossplane-runtime v0.15.1-0.20220106140106-428b7c390375
 	github.com/google/go-cmp v0.5.6
-	github.com/google/go-containerregistry v0.7.0
-	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20210330174036-3259211c1f24
+	github.com/google/go-containerregistry v0.8.1-0.20220120151853-ac864e57b117
+	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20220120151853-ac864e57b117
```

Comment:
I would typically try to stay pinned to a release version here, but I am very familiar with this dependency and feel comfortable using this incremental version to move off the problematic credentials providers.

Reply:
Is it worth a TODO/note about this?

(Side note - I realised shortly after releasing v1.6.0 that I had forgotten to release and pin crossplane-runtime.)

Reply:
I'll add one 👍🏻